services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -16,6 +27,16 @@
         }
       ]
     },
+    "UploadPartRequest": {
+      "inject": [
+        {
+          "SdkPartType": {
+            "shape": "SdkPartType",
+            "documentation": "Indicates whether the request is a last part or not. This field will not be included in the request payload.<p>It is usually not required outside of testing or advanced libraries, so most users should not need to configure it."
```

Comment:
`outside of testing or advanced libraries`  ==> I'm sorry, but I wasn't able to determine what this means.

Reply:
removed this sentence